docs/UserGuide.md

```diff
@@ -120,127 +140,221 @@ clean Graphical User Interface (GUI) for easy comprehension of expenditure and s
-   Expected: New entry added: Lunch; Date: 10-10-2022; Amount: 15.60; Tag: Food
-   ![AddCommand1](images/AddCommand1.png)
-1. Add income <br>
-   - Example: `add t/i d/Tuition a/40.00 da/10-10-2022 c/Salary`<br>
-   Expected: New entry added: Tuition; Date: 10-10-2022; Amount: 40.00; Tag: Salary
-   ![AddCommand2](images/AddCommand2.png)
-
-### Deleting entries : `delete INDEX_OF_ENTRY t/ENTRY_TYPE`
-Deletes an entry.
-1. Example: `delete 2 t/e` deletes the 2nd item on the expenditure list 
-   Expenditure list:
-   1. Movie 15.60 12 Sep 2022 c/Entertainment
-   2. Drink 1.20 12 Sep 2022 c/Food <br>
-   Expected: `Deleted Entry: Drink; Date: 12-09-2022; Amount: 1.20; Tag: Food`
-   ![DeleteCommand1](images/DeleteCommand1.png)
-1. Example: `delete 2 t/i` deletes the 2nd item on the income list
-   Income list:
-   1. Tuition 40.00 12 Sep 2022 c/Salary
-   2. Allowance 100.00 12 Sep 2022 c/Allowance <br>
-   Expected: `Deleted Entry: Allowance; Date: 12-09-2022; Amount: 100.00 Tag: Allowance`
-   ![DeleteCommand2](images/DeleteCommand2.png)
-
-### Editing entries : `edit INDEX_OF_ENTRY t/ENTRY_TYPE [d/EDITED_DESCRIPTION a/EDITED_AMOUNT da/EDITED_DATE c/EDITED_CATEGORY]`
-Edits an entry, at least **1** of the optional fields must be present.
-
-1. Edits expenditure with specified fields
-   * Example: `edit 1 t/e d/ChickenRice`
-   * The expenditure at index 1 will have its description edited <br>
-   Expected: `Edited Entry: ChickenRice; Date: 21-10-2022; Amount: 4.20; Tag: Food`
-   ![EditCommand1](images/EditCommand1.png)
-1. Edits income with specified fields
-   * Example: `edit 2 t/i a/150.00 da/22-10-2022`
-   * The income at index 2 will have its amount and date edited <br>
-   Expected: `Edited Entry: CafeSalary; Date: 22-10-2022; Amount: 150.00; Tag: Salary`
-   ![EditCommand2](images/EditCommand2.png)
+### Adding entries: `add t/ENTRY_TYPE d/DESCRIPTION a/AMOUNT da/DATE c/CATEGORY`
+
+Adds an entry to the specified list.
+
+| Input               | Explanation                                                                                  |
+|---------------------|----------------------------------------------------------------------------------------------|
+| **`t/ENTRY_TYPE`**  | An entry type refers to either `e` for "Expenditure" or `i` for "Income"                     |
+| **`d/DESCRIPTION`** | Description for the entry that you are adding                                                |
+| **`a/AMOUNT`**      | Amount of the entry, formatted to 2 decimal places                                           |
+| **`da/DATE`**       | Date where the entry is added in `dd-mm-YYYY` format, e.g. `01-01-2022` for 1st January 2022 |
+| **`c/CATEGORY`**    | [Category](#categorising-your-expenses-and-income) that the entry belongs to                 |
+
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 The default date is the current date on your computer!
 </div>
```

Comment:
I forgot to remove this, I don't think our currently implementation has this feature for the default date.